react-components/src/components/CadModelContainer/useApplyCadModelStyling.tsx

```diff
@@ -131,9 +131,12 @@ function isEqualTreeIndex(
   collectionA: TreeIndexNodeCollection,
   collectionB: TreeIndexNodeCollection
 ): boolean {
+  const setA = collectionA.getIndexSet();
+  const setB = collectionB.getIndexSet();
+
   const isEqualContent =
```

Comment:
Can we rename this to something more descriptive? `setBContainsSetA`?

Reply:
Sure